.studio/chef-server-collection

```diff
@@ -392,6 +392,76 @@ function test_knife() {
   done
 }
 
+
+document "test_ocid" <<DOC
+  test OCID against the embedded Chef server
+DOC
+function test_ocid() {
+  check_if_deployinate_started || return 1
+  bootstrap_chef_user_data || return 1
+
+  test_if_webui_key_is_patched
+  test_if_env_vars_are_configured
+}
+
+test_if_env_vars_are_configured() {
+  ocid_config_file_path="$(hab pkg path chef/oc_id)/oc_id/config/settings/production.yml"
+
+  # Read settings/production.yml into an associative array
+  declare -A config_values
+  while IFS=":" read -r key value; do
+    config_values["$key"]="${value// /}"  # Remove spaces from the value
+  done < "$ocid_config_file_path"
+
+  errors=()
+
+  if [[ "${config_values[endpoint]}" != "https://127.0.0.1:443" ]]; then
+    errors+=("Endpoint not configured properly.")
+  fi
+
+  if [[ "${config_values[superuser]}" != "pivotal" ]]; then
+    errors+=("Superuser not configured properly.")
+  fi
+
+  if [[ "${config_values[ssl_verify_mode]}" != "verify_none" ]]; then
+    errors+=("Verify mode not configured properly.")
+  fi
+
+  if [[ ${#errors[@]} -gt 0 ]]; then
+    printf '%s\n' "${errors[@]}"
+    return 1
+  fi
+}
+
+test_if_webui_key_is_patched() {
+  install_if_missing core/jq-static jq
+
+  # Extract webui_key from private-chef-secrets.json
+  ocid_secrets_file_path="$(hab pkg path chef/oc_id)/oc_id/config/private-chef-secrets.json"
+  ocid_webui_key="$(jq --arg variable_name "chef-server" -r '.[$variable_name]."webui_key"' < $ocid_secrets_file_path)"
+
+  # Create a temporary file with the ocid_webui_key content
+  echo "$ocid_webui_key" > temp_ocid_webui_key.pem
```

Comment:
Pls export a variable with value as the filename:
`export temp_ocid_webui_key_file="temp_ocid_webui_key.pem"` and use this variable instead of hardcoding the file path in multiple places.

Reply:
As the file exists at the same location we are not providing the whole path, And its been used only two times, one for diff another for deletion

Do you still think we need a variable here?